src/queries/flatRateFulfillmentMethods.js

```diff
@@ -15,7 +15,28 @@ export default async function flatRateFulfillmentMethods(context, input) {
 
   await context.validatePermissions("reaction:legacy:shippingMethods", "read", { shopId });
 
-  return Shipping.find({
-    shopId
-  });
+  // aggregate pipeline to extract fulfillment methods inside shippment
+  return {
+    collection: Shipping,
+    pipeline: [
+      {
+        $match: {
+          shopId
+        }
+      },
+      {
+        $unwind: "$methods"
+      },
+      {
+        $replaceRoot: {
+          newRoot: "$methods"
+        }
+      },
+      {
+        $set: {
+          "isEnabled": "$enabled"
+        }
```

Comment:
Remove the `$set`. We handle all field name changes using GraphQL field resolvers. In `src/resolvers/FlatRateFulfillmentMethod/index.js`, you can add a line in the exported object:

```js
isEnabled: (node) => !!node.enabled
```